### ISSUE TITLE
Equal variance test failed

### ISSUE THREAD

**Issue by zyk19981118** [drorlab/gvp-pytorch#6]
Hi, I tried to run test_equalvariance.py after configuring the environment, but the output says that all tests failed. 
The output is shown in the figure below, and the configured environment is also given.

![1659670284744](https://user-images.githubusercontent.com/53036974/182995795-2938aad2-bb5a-44b7-b20c-c8df3c0ee9b2.png)

`Package               Version
--------------------- -------------------
ase                   3.22.1
atom3d                0.2.1
biopython             1.79
cached-property       1.5.2
certifi               2022.6.15
charset-normalizer    2.0.12
click                 8.0.4
cycler                0.11.0
dataclasses           0.8
decorator             4.4.2
dill                  0.3.4
easy-parallel         0.1.6
freesasa              2.1.0
googledrivedownloader 0.4
h5py                  3.1.0
idna                  3.3
importlib-metadata    4.8.3
importlib-resources   5.4.0
isodate               0.6.1
Jinja2                3.0.3
joblib                1.1.0
kiwisolver            1.3.1
llvmlite              0.36.0
lmdb                  1.3.0
MarkupSafe            2.0.1
matplotlib            3.3.4
msgpack               1.0.4
multiprocess          0.70.12.2
networkx              2.5.1
numba                 0.53.1
numexpr               2.8.1
numpy                 1.19.4
packaging             21.3
pandas                1.1.5
pathos                0.2.8
Pillow                8.4.0
pip                   21.2.4
pox                   0.3.0
ppft                  1.6.6.4
pyparsing             3.0.9
python-dateutil       2.8.2
python-dotenv         0.20.0
python-louvain        0.16
pytz                  2022.1
rdflib                5.0.0
requests              2.27.1
scikit-learn          0.24.2
scipy                 1.5.4
setuptools            49.6.0.post20210108
six                   1.16.0
sklearn               0.0
tables                3.7.0
threadpoolctl         3.1.0
torch                 1.8.1+cu111
torch-cluster         1.5.9
torch-geometric       1.7.0
torch-scatter         2.0.6
torch-sparse          0.6.10
torchaudio            0.8.1
torchvision           0.9.1+cu111
tqdm                  4.38.0
typing_extensions     4.1.1
urllib3               1.26.10
wheel                 0.37.1
zipp                  3.6.0
`

Is it the GVP-GNN problem of the pytorch version?
